<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTCPSettingsClass</class>
 <widget class="QWidget" name="DialogTCPSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogTCPSettings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #323232;
color: #999999;
font-family: KaiTi;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>Communication Protocol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox{border:1px solid #212121;}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Protocol Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbProtocol">
          <property name="styleSheet">
           <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAddress">
          <property name="text">
           <string>Local IP Adress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditIP">
          <property name="styleSheet">
           <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPort">
          <property name="text">
           <string>Local Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPort">
          <property name="styleSheet">
           <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnConnect">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">

QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;

}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDisConnect">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;

}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>Device Debugging</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">QGroupBox{border:1px solid #212121;}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,1">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Receive Data</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btn_ReceiveClear">
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>KaiTi</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;

}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
</string>
                 </property>
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QTextEdit" name="textEditReceive">
               <property name="styleSheet">
                <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="styleSheet">
             <string notr="true">border-top: 3px solid #141414;   </string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QWidget" name="widgetLocal" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Local Host:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditLocalIP">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Local Port:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditLocalPort">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>138</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widgetConnectors" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelClient">
                  <property name="text">
                   <string>Connect Objects:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbClient">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>417</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Send Data</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnSend">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;

}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
                 </property>
                 <property name="text">
                  <string>Send</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_SendClear">
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;

}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
                 </property>
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QTextEdit" name="textEditSend">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
